scripts/exdump_monitor.sh

```diff
@@ -898,7 +898,7 @@ export DUMP_NUMBER=8
 DTIM_latest_sstvcw=+0.49
 DTIM_latest_sstvpw=+0.49
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 sstvcw sstvpw 
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 sstvcw sstvpw 
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.